tests/test_simulation.py

```diff
@@ -3810,10 +3989,23 @@ def test_averaging_per_body_nodal(
             mat_ids=bodies_in_selection,
             components=components,
             skin_mesh=res._fc[0].meshed_region,
+            additional_scoping=additional_scoping,
         )
     else:
-        # get reference data from mechanical
-        ref_data = get_ref_per_body_results_mechanical(ref_files[result], mesh)
+        # Cannot take reference for Mechanical because the named selection
```

Comment:
@janvonrickenbach so what is the reference then?

Reply:
The elemental nodal result is rescoped to the additional_scoping and then averaged. I added a comment.